### PR TITLE
Collapsible rows rework

### PR DESCRIPTION
**Pull request by PMarcL**

### CODE REVIEW

File: scss/tables/table-collapsible-rows.scss [hunk 1]
```diff
@@ -42,6 +42,15 @@ table {
           .container {
             display: none;
           }
+
+          h4 {
+            padding-top: 10px;
+
+            font-size: 12px;
```
Comment (JPLachance) [inline scss/tables/table-collapsible-rows.scss]: Why would we change the h4 default font size? Maybe we should not use h4.

File: scss/tables/table-collapsible-rows.scss [hunk 2]
```diff
@@ -42,6 +42,15 @@ table {
           .container {
             display: none;
           }
+
+          h4 {
+            padding-top: 10px;
```
Comment (JPLachance) [inline scss/tables/table-collapsible-rows.scss]: Add a variable.

File: scss/tables/table-collapsible-rows.scss (hunk 1 above)
Comment (pochretien) [inline scss/tables/table-collapsible-rows.scss]: +1 keep the default font-size for h4 or if u need less, use a other one

File: scss/tables/table-collapsible-rows.scss [hunk 3]
```diff
@@ -42,6 +42,15 @@ table {
...
+
+            font-size: 12px;
+            font-weight: bold;
+            text-transform: uppercase;
```
Comment (pochretien) [inline scss/tables/table-collapsible-rows.scss]: can u confirm with UX, in the mockup, the collapsable section title are in TitleCase and now we use Uppercase

**Comment by GermainBergeron** [comment]
run `gulp sass:format` then :shipit:

**Pull request merged**